templates/app/aspnet-core/src/MyCompanyName.MyProjectName.Domain/Data/MyProjectNameDbMigrationService.cs

```diff
@@ -108,14 +108,21 @@ private async Task SeedDataAsync(Tenant tenant = null)
             await _dataSeeder.SeedAsync(tenant?.Id);
         }
 
-        private async Task<bool> MigrationsFolderExists()
+        private async Task<bool> DbMigrationsProjectExistsAsync()
```

Comment:
It is not an asynchronous method, I think the method should be:  `private bool DbMigrationsProjectExists()`